lib/realtime.nit

```diff
@@ -10,7 +10,7 @@
 # You  are  allowed  to  redistribute it and sell it, alone or is a part of
 # another product.
 
-# Provides the Clock utility class to keep time of real time flow
+# Keep time of the real wall clock time with `Clock`
```

Comment:
I think that the synopsis should be clear if this is a library, a tool (or both)